### PR TITLE
[identd] bug fixes

### PR DESCRIPTION
**Pull request by vague666**

### CODE REVIEW

**Comment by vague666** [comment]
No idea :P Ask git. I was looking at it also and couldn't see anything new on that line

File: scripts/identd.pl [hunk 1]
```diff
@@ -61,7 +61,7 @@ sub cmd_help {
 sub start_ident_server {
   my $port = Irssi::settings_get_int('identd_port') // 8113;
   Irssi::print("Identd - starting...") if VERBOSE;
-  $ident_server = IO::Socket::INET->new( Proto => 'tcp', LocalAddr => '0.0.0.0' , LocalPort => $port, Listen => SOMAXCONN, ReusePort => 1) or print "Can't bind to port $port, $@";
+  $ident_server = IO::Socket::INET->new( Proto => 'tcp', LocalAddr => '0.0.0.0' , LocalPort => $port, Listen => SOMAXCONN, ReusePort => 1) or print "Cam't bind to port $port, $@";
```
[the remark this answers is not in the document]
Comment (vague666) [reply scripts/identd.pl:64]: No idea :P Ask git. I was looking at it also and couldn't see anything new on that line

**Comment by tdtooke** [comment]
Hi, I'm Bossierbossman on IRC.  I tried that at home and it didn't seem to change anything.  I got a nice verbose response, at some point a 'no ident response' and at the end I got a message leading me to believe it was successful but I still have the '~' so it wasn't.  Just looking at this if I had a port problem I should have received 'Can't bind to port' but that doesn't happen.  Presently I'm at work so I can't test anything but I'm usually available for this sort of thing 6:00PM to 12:00AM mon-friday on Pacific US time.  I have no idea how that translates internationally.  Thanks for all that you guys do!  It's a very good IRC client.

**Comment by vague666** [comment]
@tdtooke make sure you have `/set identd_verbose on` so you see when a connection happens and hopefully get some indication where it stops if it gets that far. Also, did you try `/set identd_strict_conn off` so the code doesn't match incoming identd connections against active irc server connection attempts

**Pull request merged**